{{ cookiecutter.format }}/{{ cookiecutter.app_name }}.wxs

```diff
@@ -38,141 +23,293 @@
         <Property Id="ARPNOREPAIR" Value="1" />
         <Property Id="ARPNOMODIFY" Value="1" />
 
-        <Upgrade Id="$(var.ProductUpgradeCode)">
-            <UpgradeVersion
-                    Minimum="$(var.ProductVersion)"
```

Comment:
The `Upgrade` element is no longer required because WiX now [handles this by default](https://docs.firegiant.com/wix/whatsnew/#default-major-upgrades).